ui/shared/components/panel/variants/VariantFamily.jsx

```diff
@@ -159,7 +162,7 @@ const VariantFamily = ({ variant, project, family, individualsByGuid }) => {
       <IndividualCell>
         <Header size="small">
           Family<HorizontalSpacer width={5} />
-          <Popup
+          <PopupWithModal
```

Comment:
didn't see this in the previous PR, but would vote for naming conventions that indicate the object type (eg. something like "VariantFamilyPanel" or "VariantFamilyInfoPanel" instead of "VariantFamily")

Reply:
okay, but that seems really unrelated to the changes in this PR so I'd rather do a separate one where we rename all the files with that convention

Reply:
sounds great

Reply:
Also, logically this would extend to renaming Family.jsx => FamilyPanel.jsx (unless there's a better word than "panel" for div/grid-type components), and most components in the 'variants' folder, so if you agree with this, we should rename a bunch of things